src/components/Filters.vue

```diff
@@ -8,10 +8,10 @@
           <b-row>
 
             <!-- Location -->
-            <b-col lg="6" sm="12">
+            <b-col lg="6" sm="12" v-if="showFilterType('location')">
```

Comment:
I think we can go further into modularity here. It is worth to create a subcomponent called FilterSelect that receives all the possible values for the select. As well as create a slot inside Filters component to where we could include all types of filters that we need. 
```
<Filters>
<FilterSelect :options="options" :filter-object="filters" filter-id="id" @change="update($event)" @search="search($event)"/>
<FilterSlider min="1" max="10" step="2" :filter-object="filters" filter-id="id" @change="update($event)"/>
</Filters>

```
`FilterSlider` is out of scope for this PR, but it is just an example. Also, there might be a possibility to access parents attributes using `this.$parent` (not tested, only read about it). This way we could avoid sending the filter-object inside each individual filter

Reply:
Couldn't find a clean way to use `$parent`, because I would need to use `this.$parent.$parent.$props.value` to traverse from a `FilterSelect` to a `Filters`.